src/main/java/core/basesyntax/service/impl/ActivitiesProcessorImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Activity;
+import core.basesyntax.service.ActivitiesProcessor;
+import java.util.List;
+
+public class ActivitiesProcessorImpl implements ActivitiesProcessor {
+    private final ActivityStrategyImpl activityStrategyImpl;
```

Comment:
As a result of empty `ActivityStrategy` interface you are required to use implementation instead of interface, direct break of DI.